tests/test-runners/run_all.py

```diff
@@ -22,6 +22,8 @@
 import os
 import sys
 
+from nuitka.tools.testing.SearchModes import SearchModeAll
```

Comment:
Here is is actually harmful, and as unused ought to have been removed. The pylint doesn't cover these files yet. I am going to fix this myself though, no big deal,as Travis doesn't run these. Maybe it ought to though, I will explore that. And also adding the test runners to pylint coverage.